en/08_Changelogs/6.0.0.md

```diff
@@ -257,6 +258,18 @@ This release includes a number of bug fixes to improve a broad range of areas. C
 
 ## API changes
 
+### GraphQL removed from the CMS {#graphql-removed}
```

Comment:
Call out early on in this section that this does _not_ impact the use of silverstripe/graphql in the front-end or with custom schemas in any capacity (unless they were using the resolvers which have new namespaces).

There's a paragraph about this at the end but I think we need to put it up front to help people not freak out.

Maybe use a `> [!INFO]` block for this so it stands out.

Reply:
Updated